frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -632,6 +632,15 @@ pub mod pallet {
 		#[pallet::constant]
 		type SignedDepositWeight: Get<BalanceOf<Self>>;
 
+		/// The number of snapshot voters to fetch per block.
+		///
+		/// In the future, this value will make more sense with multi-block snapshot.
+		///
+		/// Also, note the data type: If the voters are represented by a `u32` in `type
+		/// CompactSolution`, the same `u32` is used here to ensure bounds are respected.
+		#[pallet::constant]
+		type VoterSnapshotPerBlock: Get<CompactVoterIndexOf<Self>>;
```

Comment:
Not critical, but is there any reason not to make this a `const`?

Reply:
Not really, just to be consistent. We generally prefer `Get` over `const` because it is more easy to mutate in tests (see all the `thread_local` mumble jumble). Otherwise, in production it is just extra boilerplate.